README.md

```diff
@@ -3,3 +3,26 @@
 This action allows you to easily upload a charmed operator to [charmhub.io][charmhub].
 
 [charmhub]: https://charmhub.io/
+
+
+## Developing
+
+Start by cloning the repository:
+
+```sh
+git clone git@github.com:canonical/charmhub-upload-action.git
+cd charmhub-upload-action
+```
+
+Then, run this command to ensure you don't forget to build `dist/index.js`:
+
+    ln -s ../../hooks/pre-commit .git/hooks/pre-commit
+
+Finally, set up `ncc` to handle compiling the action into a single JavaScript file:
+
+    sudo snap install node --classic --channel 12/stable
+    mkdir ~/.npm-global
+    npm config set prefix '~/.npm-global'
+    npm install -g @vercel/ncc
```

Comment:
The use of global npm packages is brittle and almost always ends up in "works on my machine" discussions due to version mismatches. Move this to a project dev dependency and encourage the use of `npm inistall && npm run build` instead.

Reply:
Agree